backstop/tests/table.html

```diff
@@ -273,15 +273,66 @@ <h2>Default</h2>
             </div>
             <div class="iui-cell iui-slot">
               <button
-                class="iui-button iui-borderless"
+                class="iui-button iui-borderless iui-dropdown"
                 aria-label="Modify columns"
                 type="button"
+                onclick="openColumnManger()"
               >
                 <svg-column-manager
                   class="iui-button-icon"
                   aria-hidden="true"
                 ></svg-column-manager>
               </button>
+              <div
+                data-tippy-root=""
+                id="tippy-1"
```

Comment:
It doesn't make sense to emulate the tippy dom structure if you are not using tippy.js here. You can cleanup this whole thing to just use the `ul`.

Reply:
tippy structure has been removed